src/priority.py

```diff
@@ -0,0 +1,43 @@
+class Priority(object):
+    """Return."""
```

Comment:
When you are composing these data structures, do your best to use docstrings to showcase your knowledge of how the structure works. Also use the readme to compose a general overview of all of your data structures. When you have docstrings like 'Return.', this is what your class looks like when someone asks for documentation on something like iPython:

```python
In [74]: pq?
Type:           Priority
String form:    <priority.Priority object at 0x10ced3eb8>
File:           ~/Documents/code_fellows/401python-d2/sandbox/ptrompeter/data-structures/src/priority.py
Docstring:      Return.
Init docstring: Return.
```

It would be best to just have no docstring as opposed to this one.